perception/detected_object_validation/schema/object_lanelet_filter.schema.json

```
@@ -0,0 +1,121 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Object Lanelet Filter",
+  "type": "object",
+  "definitions": {
+    "object_lanelet_filter": {
+      "type": "object",
+      "properties": {
+        "filter_target_label": {
+          "type": "object",
+          "properties": {
+            "UNKNOWN": {
+              "type": "boolean",
+              "default": false,
```

@batuhanbeytekin Thank you for your PR. Did you have any intention to change the default parameter here, compared with param.yaml?
```suggestion
              "default": true,
```